examples/toolbar/toolbar.html

```diff
@@ -44,32 +46,63 @@ <h1>Toolbar Example</h1>
     <h2 id="ex_label">Example</h2>
     <div role="separator" id="ex_start_sep" aria-labelledby="ex_start_sep ex_label" aria-label="Start of"></div>
     <div id="ex1">
-        <div class="toolbar" role="toolbar">
-            <div class="toolbar-item selected" role="button" tabindex="0">
-                Tool 1
+        <div class="toolbar" role="toolbar" aria-label="Toolbar">
+          <div class="group characteristics">
+            <div role="button" class="toolbar-item bold" aria-pressed="false" value="bold" tabindex="0">
+              <i class="fas fa-bold"></i>
```

Comment:
This content does not provide an accessible label so none of the buttons in this toolbar are accessible.